discovery-provider/README.md

```diff
@@ -1,3 +1,45 @@
-An Audius Discovery Provider is a service that indexes the contents of the Audius contracts on the Ethereum blockchain, for Audius users to query. The indexed content includes user, track, and album/playlist information along with social features. The data is stored for quick access, updated on a regular interval, and made available for clients via a RESTful API.
+# Discovery Node
 
-For detailed instructions on how to develop on the service, please see [The Wiki](https://github.com/AudiusProject/audius-protocol/wiki/Discovery-Provider-‐-How-to-run#running-for-development).
\ No newline at end of file
+
+A Discovery Node is the metadata and indexing layer for the Audius Protocol.
+
+
+## Development
+
+See the dev-tools [README](https://github.com/AudiusProject/audius-protocol/tree/master/dev-tools).
+
+```bash
+audius-compose up
+
+curl http://audius-protocol-discovery-provider-1:5000/health_check
+```
+
+
+## Testing
```

Comment:
Was thinking more in the lines of

```sh
audius-compose test discovery-provider <pytest args>
```

which will take care of launching elasticsearch automatically, can make a followup pr for this

Reply:
this would be awesome, totally agree. if you can handle that follow-up, that'd be awesome. I think I understand what you're thinking though.